### ISSUE TITLE
Forms - Symbols not fitting on mobile artboards in Sketch

### ISSUE THREAD

**Issue by humancompanion-usds** [department-of-veterans-affairs/vets-design-system-documentation#1564]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

Some of our symbols seemingly no longer fit on their mobile artboards for Form components in Sketch.

## What I expected to happen

I expect symbols for mobile to fit on a 320px artboard or, if we believe that the new default size should be wider then we need to update those artboards appropriately.

## Reproducing

Steps to reproduce:

1. Open Sketch and the Pattern library
2. Look at the aforementioned list of Form components and their mobile artboards.
3. Notice how a bunch don't fit on the artboard and bleed over the right side.

## Urgency
How urgent is this request? Please select the approriate option below and/or provide details
- [ ] This bug is blocking work currently in progress
- [x] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other

### Details

Here is the list of components in Sketch that go over the edge of their artboard:
* Date input (screenshot below)
* Number input
* Textarea - Error variation (doesn't respect the gutter that is on mobile)

![Screen Shot 2023-02-17 at 5 24 00 PM](https://user-images.githubusercontent.com/52176351/219806118-500a0a43-4331-4ea5-a9bf-f75de5ae9019.png)

**Comment by LillyBoot** [comment]
Fixed the bugs. I noticed in Storybook that the Date Input in mobile view did not have a gutter. I had to guess of the sizing for the sketch file so that needs review.

**Comment by humancompanion-usds** [comment]
@LillyBoot - Unfortunately, the gutter changes depending on the page. It is 8px on most pages but then pages that use the USWDS v1 grid have 9.375px. That's most form pages. However, there are yet other places where it is 12px. So 8px or rounding up to 10px or 12px is where we landed. 

Thus for form components, I'd round up to 10px. For everything else, I'd use 8px. I don't know where we got 12px from and I'd correct those as we find them to 8px or 10px.

The other thing to track is to make sure that the gutter widths are equal on the right and left. On the error states we don't seem to be doing that consistently.
